base/main_test_cluster.go

```diff
@@ -203,3 +214,48 @@ func (c *tbpClusterV2) supportsMobileRBAC() (bool, error) {
 	}
 	return major >= 7 && minor >= 1, nil
 }
+
+// mobileXDCRCompatible checks if a cluster is mobile XDCR compatible, a cluster must be enterprise edition AND > 7.6.1
+func (c *tbpClusterV2) mobileXDCRCompatible() (bool, error) {
```

Comment:
Can we move this logic to the constructor for `tbpClusterV2` and just set a variable here, so we don't call this each time we open a bucket? I think that would make the code logic easier.